test/library/standard/DataFrames/DataFrames.chpl

```diff
@@ -1030,75 +1030,75 @@ module DataFrames {
   /*
    * ARITHMETIC AND INEQUALITY OPERATORS
    */
-  proc +(lhs: borrowed Series, rhs: borrowed Series) {
+  operator Series.+(lhs: borrowed Series, rhs: borrowed Series) {
```

Comment:
Is the rationale by not attaching all these to the `Series` type that a `Series` should always know how to operate against itself, but if you want other types then you'll need to access `DataFrames`?

Reply:
I was taking the approach of only converting operators to methods that I was certain should be associated with a particular type.  I think it's reasonable for that tactic to have been more cautious than is strictly necessary, so am okay with making more operators into methods, but will only do so when prompted.  So if you think these should also be methods, I'm fine with doing so

Reply:
It looks like in deleting duplicates, I deleted the actual comment.  Ugh.

Anyways, what I meant to say was that the initial effort was cautious and only marked cases where the type matched as an operator method.  I'm okay with making more operators into methods when prompted, so if you think they should be, then I will do so